### PR TITLE
feat(isolated-declarations): support for class function overloads

### PR DESCRIPTION
**Pull request by Dunqing**

### CODE REVIEW

**Comment by Dunqing** [comment]
* **#3811** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/3811?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Dunqing and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: napi/transform/package.json [hunk 1]
```diff
@@ -26,5 +26,6 @@
         "aarch64-apple-darwin"
       ]
     }
-  }
+  },
```
Comment (Boshen) [inline napi/transform/package.json:29]: ?

File: napi/transform/package.json
Comment (Boshen) [inline napi/transform/package.json:1]: ?

Comment (Dunqing) [reply napi/transform/package.json:1]: Ah

**Pull request merged**